style.css

```diff
@@ -0,0 +1,38 @@
+head {
+    font-size: larger;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```

Comment:
If the font is used throughout the document, it could be better to just apply it once at a higher level than in each text section